packages/common/browser/gated-download-wufoo.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="button-wrapper" v-if="canDownload">
+    <p>Your download should start automatically. If not, click the button below to access this document.</p>
+    <a :href="target" class="btn btn-lg btn-primary" target="_blank"> {{ label }}</a>
```

Comment:
- Remove the extra space before `{{ label }}`
- Add `rel="noopener norefer"` attribute since this is a `target="_blank"` link